cfd_protocol/src/protocol.rs

```diff
@@ -70,20 +71,19 @@ where
 /// * `maker_punish_params` - The punish parameters of the maker.
 /// * `taker` - The initial parameters of the taker.
 /// * `taker_punish_params` - The punish parameters of the taker.
-/// * `oracle_pk` - The public key of the oracle.
-/// * `nonce_pks` - One R-value per price digit signed by the oracle. Their order matches a big
-///   endian encoding of the price.
+/// * `oracle_pk` - The public key of the oracle. endian encoding of the price.
```

Comment:
The 2nd part of these docs seem stale?

Reply:
Oh, I didn't notice the second sentence.